data_structures/src/chain.rs

```diff
@@ -733,6 +787,21 @@ pub struct PublicKey {
     pub bytes: [u8; 32],
 }
 
+/// Public Key data structure
+#[derive(Debug, Default, Eq, PartialEq, Clone, Serialize, Deserialize)]
+pub struct SecretKey {
+    pub bytes: [u8; 32],
+}
+
+/// Public Key data structure
+#[derive(Debug, Default, Eq, PartialEq, Clone, Serialize, Deserialize)]
+pub struct ExtendedSecretKey {
+    /// Secret key
+    pub secret_key: SecretKey,
+    /// Chain code
+    pub chain_code: [u8; 32],
```

Comment:
And being paranoid, probably we should use `protected::Protected` here also

Reply:
Created issue #560